scripts/insights-core-selinux.spec

```diff
@@ -0,0 +1,56 @@
+%define distro redhat
+%global debug_package %{nil}
+%global modulename insights_core
+%global selinuxtype targeted  #FIXME: which type? mls / targeted
+
+
+Name:           insights-core-selinux
+Version:        3.5.14
+Release:        1%{?dist}
+Summary:        Insights Core SELinux policy
+
+License:        Apache-2.0
+URL:            https://github.com/RedHatInsights/insights-core-selinux
+Source0:        %{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+Requires:       selinux-policy
```

Comment:
This is not enough.
Should be:
`Requires: selinux-policy-%{selinuxtype}
Requires(post): selinux-policy-%{selinuxtype}
BuildRequires: selinux-policy-devel
%{?selinux_requires}`

Reply:
Actually, given that this package will probably use a special release protocol, we should go without "selinux_requires" and replace it with the following:
```
Requires: selinux-policy >= %{selinux_policy_version}
BuildRequires: pkgconfig(systemd)
Requires(post): selinux-policy-base >= %{selinux_policy_version}
Requires(post): libselinux-utils
Requires(post): policycoreutils
Requires(postun): libselinux-utils
Requires(postun): policycoreutils
```
This will also need a global variable which you will increment as needed 
`%global selinux_policy_version 40.13.31`
^^^ this is the latest version available in rhel-10

Reply:
Thanks for the detailed comment, I updated it.